riscv/src/small_field/runtime.rs

```diff
@@ -76,14 +76,14 @@ impl Runtime {
             [
                 r#"instr shl XL, YL, ZH, ZL, WL
                     link ~> (tmp1_h, tmp1_l) = regs.mload(0, XL, STEP)
-                    link ~> (tmp2_h, tmp2_l) = regs.mload(0, YL, STEP)
+                    link ~> (tmp2_h, tmp2_l) = regs.mload(0, YL, STEP + 1)
```

Comment:
just to understand: is it an issue having two reads in the same time step?

Reply:
For the same address actually yes (which happened earlier when I was testing the new Mock prover stuff). I also thought it wasn't a problem, and it actually didn't use to be when we used a lookup, but with a permutation it is an issue because the count doesn't match anymore.